geode-core/src/main/java/org/apache/geode/internal/lang/SystemPropertyHelper.java

```diff
@@ -113,6 +113,14 @@ public class SystemPropertyHelper {
    */
   public static final String RE_AUTHENTICATE_WAIT_TIME = "reauthenticate.wait.time";
 
+  /**
+   * Maximum number of concurrent executions in a server of wan-copy region commands.
+   * Once the maximum number is reached, subsequent executions will be halted until
+   * a thread for any of the ongoing executions is released.
+   */
+  public static final String WAN_COPY_REGION_MAX_CONCURRENT_THREADS =
+      "geode.wan.copy-region.max-threads";
```

Comment:
Raised this concern in another PR about the ongoing inconsistency in naming conventions for system properties. I don't have a good answer, just raising this again.

Reply:
@pivotal-jbarrett can you please elude to what the inconsistencies are and how you believe they should be resolved. (possibly even a link to a doc and agreed naming approach) As I had previously asked @albertogpz to change it to something more contextual and it seems believe it to be inconsistent. 
In addition you should see that I had previously (on 8 Mar) asked  yourself and @upthewaterspout to comment on the system properties naming convention.

Reply:
Here is the other PR #7517 currently under review. It uses the old `GeodeGlossary.GEMFIRE_PREFIX` prefix with `Class.CONSTANT` suffix. It also directly access System Properties directly via `Integer.getInteger`. This PR uses a hardcoded `geode` prefix and utilizes `SystemProperty.getProductIntegerProperty`. I haven't had time to investigate a better and consistent approach to naming and access. Just raising it again here some maybe someone will and post a proposal to dev@geode.